examples/geomstats_in_pymanopt.py

```diff
@@ -0,0 +1,113 @@
+"""This very simple example demonstrates how geometries from geomstats can be
+used in pymanopt to perform optimization on manifolds. It uses the Riemannian
+steepest descent solver.
+
+The example currently requires installing the pymanopt HEAD from git:
+
+    pip install git+ssh://git@github.com/pymanopt/pymanopt.git
+"""
+
+import functools
+import os
+os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"  # NOQA
```

Comment:
Do we need this? This is related to tensorflow, which is not imported?

Reply:
It's not needed, just silences an ugly warning that get's triggered since pymanopt imports tensorflow. I can remove it if it's preferable. Might otherwise give the impression it's needed for something.